app/src/main/java/com/github/orkest/Model/FireStoreDatabaseAPI.kt

```diff
@@ -144,6 +144,22 @@ class FireStoreDatabaseAPI {
 
     //===========================SONG POST OPERATIONS======================
 
+    fun storeTokenInDatabase(username:String,token:String?): CompletableFuture<Boolean>{
```

Comment:
This works as long as the token does not expire quickly. Unfortnately it is not possible in Spotify, a token expires after one hour, which is why it is not stored